packages/@aws-cdk/alexa-ask/.eslintrc.js

```diff
@@ -1,2 +1,3 @@
 const baseConfig = require('../../../tools/cdk-build-tools/config/eslintrc');
+baseConfig.parserOptions.project = __dirname + "/tsconfig.json";
```

Comment:
```suggestion
baseConfig.parserOptions.project = __dirname + '/tsconfig.json';
```
use the quotes enforced by the eslint config? 😄

Reply:
Heh, shoemaker's son always goes barefoot :)